petric.py

```diff
@@ -169,26 +169,25 @@ def get_data(srcdir=".", outdir=OUTDIR, sirf_verbosity=0):
 
 
 if SRCDIR.is_dir():
-    data_metrics_pairs = [
-        (get_data(srcdir=SRCDIR / "Siemens_mMR_NEMA_IQ", outdir=OUTDIR / "mMR_NEMA"),
+    data_dirs_metrics = [
+        (SRCDIR / "Siemens_mMR_NEMA_IQ", OUTDIR / "mMR_NEMA",
          [MetricsWithTimeout(outdir=OUTDIR / "mMR_NEMA", transverse_slice=72, coronal_slice=109)]),
-        (get_data(srcdir=SRCDIR / "NeuroLF_Hoffman_Dataset", outdir=OUTDIR / "NeuroLF_Hoffman"),
+        (SRCDIR / "NeuroLF_Hoffman_Dataset", OUTDIR / "NeuroLF_Hoffman",
          [MetricsWithTimeout(outdir=OUTDIR / "NeuroLF_Hoffman", transverse_slice=72)]),
-        (get_data(srcdir=SRCDIR / "Siemens_Vision600_thorax",
-                  outdir=OUTDIR / "Vision600_thorax"), [MetricsWithTimeout(outdir=OUTDIR / "Vision600_thorax")])]
+        (SRCDIR / "Siemens_Vision600_thorax", OUTDIR / "Vision600_thorax",
+         [MetricsWithTimeout(outdir=OUTDIR / "Vision600_thorax")])]
 else:
     log.warning("Source directory does not exist: %s", SRCDIR)
-    data_metrics_pairs = [(None, [])]
-# first dataset
-data, metrics = data_metrics_pairs[0]
+    data_dirs_metric = [(None, None, [])]
 
 if __name__ == "__main__":
```

Comment:
```suggestion
if __name__ != "__main__":
    srcdir, outdir, metrics = data_dirs_metrics[0]
    data = get_data(srcdir=srcdir, outdir=outdir)
    metrics[0].reset()
else:
```

Reply:
why do we need this?

Reply:
It's documented in the readme & main.py files (to keep docs simple)